components/ingest-service/server/job_scheduler_server.go

```diff
@@ -65,15 +65,68 @@ func (server *JobSchedulerServer) GetStatusJobScheduler(ctx context.Context,
 			j.LastElapsed = sched.LastEnd.Sub(*sched.LastStart).String()
 		}
 
-		jobs = append(jobs, &j)
+		return &j, nil
+	}
+
+	var (
+		jobs      = []*ingest.Job{}
+		jobStatus = &ingest.JobSchedulerStatus{}
+	)
+	for _, sched := range schedules {
+		switch sched.WorkflowName {
+		case "purge_timeseries":
+			var (
+				purgePolicies []purge.Policy
+				job           *ingest.Job
+			)
+
+			if err = sched.GetParameters(&purgePolicies); err != nil {
+				return jobStatus, status.Error(codes.Internal, err.Error())
+			}
+
+			toJobName := func(index string) string {
+				index = strings.ReplaceAll(index, "-", "_")
+				return fmt.Sprintf("purge_timeseries_%s", index)
+			}
+
+			for _, policy := range purgePolicies {
+				job, err = asIngestJob(
+					sched,
+					toJobName(policy.IndexName),
+					fmt.Sprintf("%dd", policy.OlderThanDays),
+				)
+
+				if err != nil {
+					return jobStatus, status.Error(codes.Internal, err.Error())
+				}
+				jobs = append(jobs, job)
+			}
+		// SingleJobWorkflow's
+		default:
+			var (
+				threshold string
+				job       *ingest.Job
+			)
+
+			if err = sched.GetParameters(&threshold); err != nil {
+				return jobStatus, status.Error(codes.Internal, err.Error())
+			}
+
+			job, err = asIngestJob(sched, sched.WorkflowName, threshold)
+			if err != nil {
+				return jobStatus, status.Error(codes.Internal, err.Error())
+			}
+			jobs = append(jobs, job)
+		}
 	}
 
-	return &ingest.JobSchedulerStatus{
-		Jobs:    jobs,
-		Running: true,
-	}, nil
+	jobStatus.Running = true
+	jobStatus.Jobs = jobs
+
+	return jobStatus, nil
 }
 
+// FIXME: runJobNow hardcodes shit that it should not
```

Comment:
Yikes! This seems like a pretty major bug that we probably want to fix in a separate PR.

Reply:
Submitted this as a fix here: https://github.com/chef/automate/pull/1223